.gitignore

```diff
@@ -6,3 +6,4 @@ npm-debug.*
 .expo/
 *.sqlite3
 .DS_Store
+jsconfig.json
```

Comment:
Do we still need this ignore?

Reply:
This is not related to this PR. But in some IDE's (for e.g. VS Code) if you have enabled `jshint` extension this file is added by default to the root folder. This is just for safe keeping

Reply:
OK